FlorenceBML/BMLacq384/BMLacq384.xml

```diff
@@ -30,12 +30,217 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                                    </explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.5">
+                                        <locus from="25v" to="26v"/>
+                                        <title>Prayer</title>
+                                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡</hi><gap reason="ellipsis" unit="lines" quantity="2"/>
+                                            ጸሎት፡ ዘመንገደ፡ ሰማይ፡</incipit>
+                                        <explicit xml:lang="gez">ሰአሉ፡ ለነ፡ ሰላትያል፡ ወስዳክያል፡ እክናታኤል፡ ሰአሉ፡ ለነ፡ አስተምህሩ፡ ለነ፡ ቅ<pb n="26v"/>ድመ፡ መንበሩ፡ ለእግዚእነ።
+                                            ወሊተኒ፡ አድኅነኒ፤ ለዓመትክ፡ <hi rend="rubric"><persName ref="PRS14219WalattaIyasus">ወለተ፡ ኢያሱስ፡</persName></hi></explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.6">
+                                        <locus from="26v" to="27v"/>
+                                        <title>Prayer</title>
+                                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡</hi><gap reason="ellipsis" unit="lines" quantity="2"/>
+                                            ጸሎት፡ ዘመንገደ፡ <pb n="27r"/>ሰማይ፡</incipit>
+                                        <explicit xml:lang="gez">በጣፃዌ፡ ወበልንጰዌ፡ ስሙ፡ ለመድኃኒነ፡ ወበ<hi rend="rubric">ማርያም፡</hi> ወላዲትከ፡ በጥ፡ 
+                                            በ<pb n="27v"/>ርያዶስ፡ ዘንጹሕ፡ ጽርሐ፡ መቅደሱ፡ አልቦ፡ ዘየአምሮሙ፡ ለአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ዕቀበኒ፡ ለዓመትከ፡ 
+                                            <persName ref="PRS14219WalattaIyasus">ወለተ፡ <hi rend="rubric">ኢያሱስ፡</hi></persName></explicit>
+                                    </msItem>
+                                    <msItem xml:id="ms_i1.2.7">
+                                        <locus from="27v" to="29r"/>
+                                        <title>Prayer</title>
+                                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ ወአ<pb n="28r"/>ብ፡</hi><gap reason="ellipsis" unit="lines" 
+                                            quantity="2"/> ጸሎት፡ ዘመንገደ፡ ሰማይ፡</incipit>
+                                        <explicit xml:lang="gez"> በኤ<pb n="28v"/>ልዳን፡ ስምከ፡ ወበምኅንሳ፡ ማኅደርከ፡ በ፲ወ፭፡ ነቢያት፡ በ፲ወ፪ሐዋርያት፡ በ፸ወ፪አርድእት፡ በ፭፻ቢጽ፡
+                                            በ፫፻፲ወ፰ርቱዓነ፡ ሃይማኖት፡ <pb n="29r"/>በ፳ወ፬ካህናተ፡ ሰማይ፡ ወበ፬እንስሳ፡ ፀዋርያነ፤ መንበሩ፡ ለእግዚአብሔር፤ ተማኅፀንኩ፡ አነ፡ ዓመትከ፡
+                                            <persName ref="PRS14219WalattaIyasus">ወለተ፡ ኢያሱ<hi rend="rubric">ስ፡</hi></persName></explicit>
+                                    </msItem>
+                                </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="29r" to="31v"/>
+                            <title>Narration of the Passion of Christ revealed to Maria Magdalene, Sara and Salome.</title>
```

Comment:
I found LIT5020Hemamatihu. It is attested in BLorient631 on f. 2v with a similar incipit: ዝንቱ፡ ሕማማቲሁ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ መድኅነ፡ ዓለም፡ ዘለሊሁ፡ ከሠተ፡ ሎን፡ ለሥላስ፡ ቅድስት፡ አንስት። እለ፡ ይሰመያ፡ ቅድስት፡ መግደላዊት፡ ወቅድስት፡ ሰሎሜ፡ ወቅድስት፡ ኤልሳቤጥ። 
However, Sara is replaced by Elisabeth here as the third witness of the resurrection. It can be the same work or a similar text. 

Beyond this witness, there are two more in the British library. Unfortunately, I cannot check any of them at the moment, because the server of the British Library is down. Do you think, the common features are enough to assume this to be the same text?

Reply:
I wrote an issue for this question: [https://github.com/BetaMasaheft/Documentation/issues/2445](url)